Makefile

```diff
@@ -156,6 +156,9 @@ test-e2e-ansible-molecule: image-build-ansible
 test-e2e-helm: image-build-helm
 	./hack/tests/e2e-helm.sh
 
+test-integration:
```

Comment:
Is not it just about the integration with OLM? WDYT about we add olm in the name to make it clear?

Reply:
We can always add more tests to `test-integration` later. We shouldn't have separate targets for every integration test IMO. The Makefile will explode more than it already has.